### PR TITLE
Most-read size reduction for STY pages

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Related to https://github.com/bbc/psammead/issues/3320

**Overall change:**
Bumps `psammead-most-read` to 4.1.0 and adds relevant props to most-read container and the STY page.

**Code changes:**

- Bumped `psammead-most-read` to 4.1.0.
- Added `size` prop to the most-read container.
- Added `size="small"` passed into the most-read container on the STY page.
- Updated Jest snapshots.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by thekp — APPROVED** [review]

File: scripts/simorghPages.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const generateLinks = (service, env, domain) => {
   return output.join('<br/>');
 };
 
-const generateLaunchDates = (service) => {
+const generateLaunchDates = service => {
```
Comment (andrewscfc) [inline scripts/simorghPages.js:67]: Bit confused why this change is needed on this PR? Won't latest be failing the prettier check if we need to make this change?

**Review by andrewscfc — COMMENTED** [review]
One of our snapshots seems to have lost some bottom padding, I'd imagine that's actually an improvement but not certain
https://www.chromatic.com/snapshot?appId=5d28eb3fe163f6002046d6fa&id=5ea9ac03f6e1c30022972fa5

Does the spacing look at bit weird between 4 & 5 and 9 & 10 for chrome here: 
https://www.chromatic.com/snapshot?appId=5d28eb3fe163f6002046d6fa&id=5ea9ac03f6e1c30022972fe4
image

I think it's because 4 and 9 are single line headlines, maybe not one for this PR but I'd imagine you'd want consistent spacing between the numbers, regardless of the number of lines the headline occupies

Looking great on the Story page 🎉 
https://www.chromatic.com/snapshot?appId=5d28eb3fe163f6002046d6fa&id=5ea9ac03f6e1c30022972fff

File: scripts/simorghPages.js (hunk 1 above)
Comment (HarveyPeachey) [reply scripts/simorghPages.js:67]: I've seen this before where my es-lint on Atom has picked this file up before. It wouldn't let me push without a `-no-verify` unless I fixed it. So I can remove it and create a separate issue for it. Not sure why this check isn't failing on latest 🤷‍♂️.

Comment (andrewscfc) [reply scripts/simorghPages.js:67]: No big deal @HarveyPeachey , just didn't make any sense was all. Feel free to leave it in the PR imo.

Comment (HarveyPeachey) [reply scripts/simorghPages.js:67]: In regards to the chromatic snapshots. Just checked the bottom padding snapshot and checked both build snapshot canvas's and the sizes are the same, so I'm assuming chromatic bugged out generating the image.

The spacing does look weird, but this existed before the PR. I agree all items should share the same height so it doesn't look skewed like that. Maybe it was a design choice to get the component out faster but not sure.

Comment (andrewscfc) [reply scripts/simorghPages.js:67]: > Just checked the bottom padding snapshot and checked both build snapshot canvas's and the sizes are the same

Cool thanks for checking I'll approve on that basis 👍 

Could you show a UXer the spacing anamoly and raise an issue if they think it's wrong?

**Review by andrewscfc — APPROVED** [review]
Approving as @HarveyPeachey has checked the snapshot issues.

File: scripts/simorghPages.js (hunk 1 above)
Comment (HarveyPeachey) [reply scripts/simorghPages.js:67]: Yeah, good shout. I'll also ask the people who worked on this in the home pod.

**Comment by thekp** [comment]
> One of our snapshots seems to have lost some bottom padding, I'd imagine that's actually an improvement but not certain

Nice spot @andrewscfc 

I believe the issue is happening because we do not have padding around the number and have it on the text instead. I'll create an issue so that we can investigate this 👍 https://github.com/bbc/psammead/issues/3437

image

**Comment by paruchurisilpa** [comment]
Looks good to me..

**Pull request merged**